README.md

```diff
@@ -36,6 +36,7 @@ The conference dataset hosted in this repository tracks the following informatio
 | [Øredev](https://oredev.org/) | [Öredev AB](https://oredev.org) | No | No | No | Never | 
 | [Lambda World](https://www.lambda.world/) | [Xebia Functional](https://www.47deg.com/) | Yes | No | Yes | Never | 
 | [GOTO](https://gotopia.tech/) | [Trifork](https://trifork.com/) | Yes | No | Reasonable  | Never | 
+| [Scala.IO](https://scala.io) | Scala Contributers | Yes | Yes | Partial | Never |
```

Comment:
I don't know what to say, is it a joke?

Reply:
Still on of the conference organiser, and those values, are the valid ones.
```
[Scala.IO](https://scala.io) | Scala.IO | "Would prefer not to" | Yes | "Full, to the capacity of the funds" | Never |
```

I was clear about it. @jdegoes there is a reason for it, and you are overstepping my/our boundaries, can you remove the bounty on Scala.IO? 

I am okay you don't want those value (out of range), but it's not okay to spread misinformation about the conference.

Reply:
There are outstanding bounties for several conferences. Feel free to claim the bounty by submitting correct information (including Unknown if information is unknown).

 I have zero tolerance toward inaccurate data so if you see any inaccuracies, let me know and it will be dealt with straight away!